manager/ui/hawtio/plugins/api-manager/ts/forms/edit-gateway.ts

```diff
@@ -89,6 +89,7 @@ module Apiman {
                 $scope.updateButton.state = 'in-progress';
                 var gateway = Gateway();
                 ApimanSvcs.update({ entityType: 'gateways', secondaryType: $scope.gateway.id }, gateway, function() {
+                    $rootScope.isDirty = false;
```

Comment:
I think that's the only place.  But only because many of the other "edit"
pages don't implement isDirty functionality at all.  :)  We can add that to
the ever growing list of improvements to make.

On Wed, Sep 30, 2015 at 11:14 AM, Rachel <notifications@github.com> wrote:

> In manager/ui/hawtio/plugins/api-manager/ts/forms/edit-gateway.ts
> <https://github.com/apiman/apiman/pull/229#discussion_r40806430>:
>
> > @@ -89,6 +89,7 @@ module Apiman {
> >                  $scope.updateButton.state = 'in-progress';
> >                  var gateway = Gateway();
> >                  ApimanSvcs.update({ entityType: 'gateways', secondaryType: $scope.gateway.id }, gateway, function() {
> > +                    $rootScope.isDirty = false;
>
> @EricWittmann <https://github.com/EricWittmann> - Thanks for tagging me
> on this. That was a good catch, I hope we don't have that issue in other
> areas. I'll do some additional testing later today. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiman/apiman/pull/229/files#r40806430>.
>